src/plugins/intel_gpu/src/kernel_selector/cl_kernels/include/batch_headers/fetch_weights.cl

```diff
@@ -309,19 +309,25 @@ inline uint get_g_os_is_zyx_osv_isv_index(uint g, uint o, uint i, uint z, uint y
 inline uint get_os_is_zyx_isv8_osv16_isv2_index(uint g, uint o, uint i,  uint z, uint y, uint x, uint x_size, uint y_size, uint z_size,
                                                       uint g_size, uint o_size, uint i_size, uint offset)
 {
-    const uint group_offset = g * o_size * i_size * z_size * y_size * x_size;
-    const uint xyz_offset = (x + y * x_size + z * x_size * y_size)* 8*16*2;
-
-    const uint i2_val = i % 2;
-    const uint i2_slice = i / 2;
-    const uint i8_v = i2_slice % 8;
-    const uint i8_s = i2_slice / 8;
+    const uint isv2_idx = i % 2;
+    const uint osv_idx = o % 16;
+    const uint isv1_idx = (i / 2) % 8;
```

Comment:
nit: isn't it `isv8_idx`?

Reply:
It means dimension not size. Implicitly all weights conversion functions follow this rule.